bin/test/i18n/test_i18n_script_utils.rb

```diff
@@ -120,6 +103,47 @@ def test_to_js_locale_returns_formated_js_locale
     end
   end
 
+  describe '.log_error' do
+    let(:log_error) {described_class.log_error(error_class, error_message)}
+
+    let(:error_class) {'expected_error_class'}
+    let(:error_message) {'expected_error_message'}
+
+    it 'returns error class and message' do
+      described_class.expects(:puts).with('[expected_error_class] expected_error_message').once
+      _(log_error)
+    end
+  end
+
+  describe '.fix_yml_file' do
+    let(:fix_yml_file) {described_class.fix_yml_file(provided_yaml_file_path)}
+
+    let(:provided_yaml_file_path) {'provided_yaml_file_path'}
+    let(:provided_yaml_file_data) {"---\nen-US:\n  data\n"}
+    let(:expected_yaml_output) {%Q["en-US":\n  data\n]}
+
+    before do
+      FileUtils.mkdir_p File.dirname(provided_yaml_file_path)
```

Comment:
let's avoid actual disk access and stub out the file write. see my comment below

Reply:
We use temporary (fake) dir for these tests:
https://github.com/code-dot-org/code-dot-org/blob/9ce543143363aa180b228392ee32d4c007a20b83/bin/test/i18n/test_i18n_script_utils.rb#L7-L9

Reply:
ah good call. yep. thank you.